README.md

```diff
@@ -187,6 +187,9 @@ A list of companies that have paid Developer Community Writer Programs.
 - [Paperspace](https://blog.paperspace.com/write-for-paperspace/) - $200-$300 per piece
   > Get paid to write articles about machine learning, data science, and more.
 
+- [Plural](https://www.plural.sh/blog/plurals-content-contribution-program/) - $300 per 1,000 word article
+  > Get paid to write articles on, what you have built with Plural, application-specific deployment guides, and architectural comparisons of popular open-     source tools against each other.
```

Comment:
there seems to be an extra space here between the dash and "source" - could you look at that? happy to merge after fixing

Reply:
I'm interested

Reply:
hey @malgamves, this is fixed 🙂